university-of-lincoln-harvard.csl

```diff
@@ -223,31 +220,27 @@
           </group>
         </else-if>
         <else-if type="book graphic report" match="any">
-          <group delimiter=". " prefix=" ">
+          <group prefix=" " delimiter=". ">
             <text variable="note"/>
```

Comment:
I know this isn't yours, but we want the note removed here. It makes no sense.

Reply:
Will remove this.